Source/GSXib5KeyedUnarchiver.m

```diff
@@ -3226,6 +3237,10 @@ - (BOOL) containsValueForKey: (NSString *)key
         {
           hasValue = [currentElement attributeForKey: @"selectedItem"] != nil;
         }
+      else if ([@"NSPriority" isEqualToString: key])
```

Comment:
This check should not be needed. The decodes should do this automatically. If it doesn't work please report back to me.

Reply:
Will do.

Reply:
Removed, works properly.